README.md

```diff
@@ -1235,6 +1235,42 @@ mockkObject(ObjBeingMocked) {
 }
 ```
 
+### Suppressing superclass calls
+
+To suppress a method call, especially a `super` call inside an overridden method, you can stub its behavior on a `spyk`.
+Using `every { ... } just runs` replaces the entire method body, preventing the original code from executing.
+
+This is particularly useful for users coming from frameworks like PowerMockito or for testing classes like Android Activities.
```

Comment:
How about adding a link to the documentation for PowerMockito's suppress here? (https://github.com/powermock/powermock/wiki/Suppress-Unwanted-Behavior)